src/builder/Actions.sol

```diff
@@ -161,6 +212,8 @@ library Actions {
         address comet;
         uint256 price;
         address token;
+        address morpho;
+        bytes32 morphoMarketId;
```

Comment:
It's unclear to me that we want to share the same action context for Comet and Morpho. Could you ask 🏦/🥬 since the client is the main consumer of action contexts?

Reply:
Will check with 🏦 and 🥬

Was thinking we could use the same context, as those are essentially the same actions.